content/v2.0/influx-templates/_index.md

```diff
@@ -12,4 +14,32 @@ from dashboards and Telegraf configurations to notifications and alerts in a sin
 Use Influx Templates to quickly set up a fresh instance of InfluxDB, backup your
 dashboard configuration, or share your configuration with the InfluxData community.
 
+## Template manifests
+A template manifest is a file that defines the contents of a template.
+Manifest files support the following formats:
+
+- YAML
+- JSON
+- Jsonnet
+
+_See [Create an Influx Template](/v2.0/influx-templates/create/) for information about
+generating manifest files._
+
+### Template resources
+Template **resources** are elements of the InfluxDB user interface (UI) that can
+be included in a template.
+The following template resources are available:
```

Comment:
## Template resources/elements
Add the following InfluxDB elements/resources (is one preferable?) to your template: